client/app/app.vue

```diff
@@ -1,13 +1,77 @@
 <template lang="pug">
     div(role="application")
-        ul
-            li: router-link(to="/foo") foo
-            li: router-link(to="/bar") bar
+        aside(role="navigation")
```

Comment:
I think the `[role=navigation]` attribute should be on the `ul` above instead of the `aside` element (mainly because the `aside` contains a `h4` which isn't a navigational item)